arrows/core/read_object_track_set_kw18.cxx

```diff
@@ -227,6 +229,17 @@ ::read_all()
     vital::time_usec_t frame_time = atof( col[COL_TIME].c_str() );
     int track_index = atoi( col[COL_ID].c_str() );
 
+
+    vital::vector_2d tracking_plane_loc = {atof(col[COL_LOC_X].c_str()),
+    				           										 atof(col[COL_LOC_Y].c_str())};
+
+    vital::vector_2d velocity = {atof(col[COL_VEL_X].c_str()),
+    			         							 atof(col[COL_VEL_Y].c_str())};
+
+    vital::vector_3d world_loc_xyz = {atof(col[COL_WORLD_X].c_str()),
+    			              							atof(col[COL_WORLD_Y].c_str()),
```

Comment:
Looks like weird spacing / line endings here.

Reply:
Yeah that was interesting, should be fixed now.